spinbot/event/label_issue_comment_event_handler.py

```diff
@@ -0,0 +1,26 @@
+from gh import AddLabel
+from .handler import Handler
+from .command import GetCommands
+from .issue_event import GetIssue
+
+class LabelIssueCommentEventHandler(Handler):
+    def __init__(self):
+        super().__init__()
```

Comment:
Is `super()` python3 style? I'm not familiar with that that does.

Reply:
Yup, it just calls the parent class's implementation of the requested method.